.travis/tests.sh

```diff
@@ -1,10 +1,12 @@
 #!/bin/bash
 
+set -ve
+
 if [[ ${TEST} = 'main' ]]
 then
 
     test_syntax() {
-        ansible-playbook -i .travis/environments/travis/inventory.ini ansible/deploy_stack.yml --syntax-check
+        ansible-playbook -i .travis/environments/travis/inventory.ini src/commcare_cloud/ansible/deploy_stack.yml --syntax-check
```

Comment:
The test failure is that now we rely on `commcare-cloud` putting virtualenv's `site-packages/.ansible` on the ansible roles path. I think we can fix this and make it more future-proof by following the pattern from below and running

```
        cp .travis/environments/travis/private.yml .travis/environments/travis/vault.yml
        COMMCARE_CLOUD_ENVIRONMENTS=.travis/environments commcare-cloud travis deploy-stack --branch=FETCH_HEAD  --skip-check --quiet --syntax-check
```

or alternatively setting this environment variable manually before the current ansible-playbook command

```
ANSIBLE_ROLES_PATH=$VIRTUAL_ENV/lib/python2.7/site-packages/.ansible ansible-playbook ...
```
(I like the first option better